ground/gcs/src/plugins/uavsettingsimportexport/uavsettingsimportexportfactory.cpp

```diff
@@ -404,7 +265,6 @@ QString UAVSettingsImportExportFactory::createXMLDocument(const enum storedData
     // create settings and/or data elements
     QDomElement settings = doc.createElement("settings");
     QDomElement data = doc.createElement("data");
-    QDomElement waypoints = doc.createElement("waypoints");
```

Comment:
Ok. So just to be clear we had _three_ waypoint representations? This was adding it to a standard settings export too, and that is what you are removing?